### PR TITLE
[READY] better contraband crates

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
feature: NT reports that the local contraband market has expanded, experts say that the Syndicate is involved.
/:cl:

contraband crates have always been really bad and underwhelming, spiced them up

### CODE REVIEW

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -157,4 +157,26 @@
...
+
+
+/obj/item/reagent_containers/pill/lsd
+	name = "hallucinogen pill"
+	list_reagents = list("mushroomhallucinogen" = 15, "mindbreaker" = 15)
+
+
+/obj/item/reagent_containers/pill/aransep
+	name = "speedy pill"
+	list_reagents = list("aransep" = 10)
```
Comment (XDTM) [inline code/modules/reagents/reagent_containers/pill.dm]: aransep

File: code/game/objects/items/storage/firstaid.dm [hunk 2]
```diff
@@ -231,3 +231,35 @@
 	new /obj/item/reagent_containers/pill/patch/silver_sulf(src)
 	for(var/i in 1 to 3)
 		new /obj/item/reagent_containers/pill/patch/styptic(src)
+
+/obj/item/storage/pill_bottle/zoom
+	name = "suspicious pill bottle"
+	desc = "The label is pretty old and almost unreadable, you recognize some chemical coumpounds."
+
+/obj/item/storage/pill_bottle/stimulant/PopulateContents()
```
Comment (francinum) [inline code/game/objects/items/storage/firstaid.dm]: All of these have the same path.

**Comment by francinum** [comment]
It might be better to tree the bottles under a new `/obj/item/storage/pill_bottle/random/` type.

File: code/game/objects/items/storage/firstaid.dm [hunk 3]
```diff
@@ -231,3 +231,35 @@
...
+/obj/item/storage/pill_bottle/zoom
+	name = "suspicious pill bottle"
+	desc = "The label is pretty old and almost unreadable, you recognize some chemical coumpounds."
+
+/obj/item/storage/pill_bottle/stimulant/PopulateContents()
+	for(var/i in 1 to 5)
+		new /obj/item/reagent_containers/pill/zoom(src)
+
+/obj/item/storage/pill_bottle/happy
+	name = "suspicious  pill bottle"
```
Comment (francinum) [inline code/game/objects/items/storage/firstaid.dm]: Every pill bottle but zoom has a strange double-space in the name.

**Review by francinum — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/pill.dm [hunk 4]
```diff
@@ -157,4 +157,26 @@
...
+
+
+
```
Comment (praisenarsie) [inline code/modules/reagents/reagent_containers/pill.dm]: wHITE SPACE

**Comment by improvedname** [comment]
Lets see...

>Omnizine ciggaretes(the ones used by traitors/nuke ops))
>Syndicate mask((the ones used by nuke ops))
>tacticool turtleneck((the ones used by nuke ops))

GEE I WONDER WHAT THE THING HES STRIVING HERE FOR

>white cannabis leaf(i recall that one being a good mutation from cannabis also a cheap way for botany to get them without any effort same applies for rainbow which can be pretty rare)

**Comment by improvedname** [comment]
If you want to spice contrabrand crates just make a list of contrabrand items and basically make them like null crates but for contrabrand the real suprise is opening your bad boy crates

**Comment by Cyprex** [comment]
Rainbow cannabis only has mindbreaker toxin, lipolicide and possibly space drugs, while cannabis/white is Lifeweed, which has 35% omnizine production, which is pretty good.
Stimulant pills are absolutily ridiculous, remove them.

**Comment by GuppyLaxx** [comment]
@improvedname
`>`TACTICOOL, THE ONE USED BY NUKE OPS
yes the ones with zero armor and no suit sensors lmao
`>`HOLY FUCKIGN SHIT SOME FUCKING CIGS AND A SYNDIE MASK!?!?!!?!?!?!
that's so fucking op holy fuckkkkkkkkkk literally nerf the PR right now because you get some shit healing cigs and an edgy gas mask
`>`JESUS FUCK NOT SOME WEED, HE PUT SOME WEED IN THE CRATE?????? WTF OP
wowee, some weed, that's so fucking good dude im gonna fucking cream myself rn i get to be high and start starving and holy shit i can make moar weed with my weed now this is so fucking amazing i totally couldnt do this in less than 5 minutes at botany bro
@Cyprex 
1u of meth isn't anything worthwhile apart from a fast method of getting maybe 10 brain damage, everything else in the pill is something any greyshit with a brain can make with literally nothing in way less than getting this crate. if you start chugging down pills you're risking it really hard as meth currently absolutely rapes your brain with mannitol.
aranesp pill is whatever, atleast that drug is finally gonna see some use

**Comment by Tlaltecuhtli** [comment]
it is a list and it picks random shit from it

**Comment by improvedname** [comment]
Then why increase the contents kek

**Comment by Tlaltecuhtli** [comment]
because there is more stuff?

**Comment by improvedname** [comment]
increase null crate contents because theres more stuff lol

**Comment by optimumtact** [comment]
this crate should be removed, the whole idea of hacking the comms console is laughable because it's trivial to do which is why the rewards suck.

remove the feature

**Comment by ExcessiveUseOfCobblestone** [comment]
@optimumtact or make it more difficult and scale the rewards appropriately ;)

**Comment by optimumtact** [comment]
There's no way you can make it more difficult that isn't trivially defeated by people knowing how to do it

**Comment by optimumtact** [comment]
good luck

**Pull request merged**